Assets/BowAndArrowImproved/Scripts/Interfaces/IDataLoader.cs

```diff
@@ -0,0 +1,4 @@
+﻿public interface IDataLoader
+{
+    public T LoadData<T>(string asset);
+}
```

Comment:
let's combine this interface and the `IDataSaver` interface into a single interface called `DataHandler` which will contain two functions: `LoadData` and `SaveData`